README.md

```diff
@@ -1,2 +1,50 @@
-# folio-deployment
-Deploy Folio to our Infrastructure
+# okube 
+An Okapi deployment toolkit.
+
+Automation for deploying a Okapi instance and its modules to a Kubernetes (k8s) cluster from a command line interface. The easy button is here.
+
+
+## Why okube?
+
+Successfully standing up Okapi and its modules into a cluster requires that certain things that we are calling 'task' happen in a specific order. Doing so manually requires that the opertator remembers the list of task and their execution order which can be error-proned. In addition, this process can take a considerable amount of time as the operator needs to wait for a task to complete before proceeding to the next. Unfortuately task execution order is not the only knowledge needed for a deployment. 
+
+Kubernetes presents terminology, tools, and techniques specific to Kubernetes that need to be learned before deploying to a cluster. This can be a tall order as Kubernetes and Devops in general cover a large surface area. Fortuately this `okube` toolkit eleviates the need to learn Okapi task execution order, Kubernetes, and Devops straight out the gate.
+
+Deploying Okapi and the modules to a new Kubernetes cluster just takes running a command such as `bin/okube deploy --environment sandbox`.
+
+## Pre-deploy Caveat
+
+Currently this toolkit is hard coded for use with Frontside's Google Container Engine (GKE) instances but can be tailored for use with another GKE instance wanting to deploy Okapi and the modules.
+
+## Deploying Cluster
+To run any commands you can `cd` (change directory) in the `scripts` folder from the root of these repo.
```

Comment:
this has actually changed slightly.  instead of running the `bin/okube` shell script that essentially wrapped the `okube` (now `okubi`) executable in a `bundle exec`, we now just recommend using bundle's binstubs functionality to install the package.  we added a Gemfile to the top level directory that just calls in `okubi` and `okapi`.
```
bundle --binstubs=.binstubs --force
```
or simply
```
bundle --binstubs
```
should be suitable for most people.  that exposes both `okubi` and `okapi` for use anywhere on the system,  although it's probably still best to run it from somewhere within the project directory.